README.md

```diff
@@ -136,6 +136,12 @@ This example just streams the body of the request. The body of the response can
 
 Check out the `examples` folder how your server could look like.
 
+The first parameter (often written as `RequestInterface $request` in examples and tests) 
+of the callback function and a middleware is a `ServerRequest` object based on the 
```

Comment:
Go ahead :-)

In my understanding we don't even need to mention the `ServerRequest` implementation detail at all, so I'm curious what your suggestion looks like :+1: